evm/src/executor/abi/mod.rs

```diff
@@ -41,6 +41,20 @@ ethers::contract::abigen!(
             envBytes32(string,string)(bytes32[])
             envString(string,string)(string[])
             envBytes(string,string)(bytes[])
+            envWithDefault(string,bool)(bool)
```

Comment:
I'd like to suggest an alternative name here:

`envOr` or even variants for all types like `envStringOr`

imo `Default` is not really fitting here

Reply:
I'm fine with `envOr`, just originally chose `envWithDefault` cause it felt more explicit, though `envOr` is more aligned with rust-like syntax